{"name": "test_web", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument\n  (Session info: chrome=100.0.4896.60)\nStacktrace:\n#0 0x5580e83ce013 <unknown>\n#1 0x5580e813089f <unknown>\n#2 0x5580e811c388 <unknown>\n#3 0x5580e811ad52 <unknown>\n#4 0x5580e811af32 <unknown>\n#5 0x5580e81324ca <unknown>\n#6 0x5580e81982ef <unknown>\n#7 0x5580e8184932 <unknown>\n#8 0x5580e8197b94 <unknown>\n#9 0x5580e8184823 <unknown>\n#10 0x5580e815a08a <unknown>\n#11 0x5580e815b1e5 <unknown>\n#12 0x5580e83fbe5d <unknown>\n#13 0x5580e8415061 <unknown>\n#14 0x5580e83fdbb5 <unknown>\n#15 0x5580e8415e08 <unknown>\n#16 0x5580e83f12f0 <unknown>\n#17 0x5580e8431988 <unknown>\n#18 0x5580e8431b08 <unknown>\n#19 0x5580e844b5dd <unknown>\n#20 0x7f4a3fe17609 <unknown>", "trace": "def test_web():\n>       browser.open('/automation-practice-form')\n\ntest_home.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:1155: in open\n    self.config._executor.get_url(relative_or_absolute_url)\n..\\venv\\lib\\site-packages\\selene\\core\\configuration.py:301: in get_url\n    self.config._driver_get_url_strategy(self.config)(url)\n..\\venv\\lib\\site-packages\\selene\\core\\configuration.py:253: in get\n    driver.get(url)\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x0000019C9DE675B0>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"invalid argument\",\"message\":\"invalid argument\\\\n  (Session info: chrome=1...\\\\n#18 0x5580e8431b08 \\\\u003Cunknown>\\\\n#19 0x5580e844b5dd \\\\u003Cunknown>\\\\n#20 0x7f4a3fe17609 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.InvalidArgumentException: Message: invalid argument\nE         (Session info: chrome=100.0.4896.60)\nE       Stacktrace:\nE       #0 0x5580e83ce013 <unknown>\nE       #1 0x5580e813089f <unknown>\nE       #2 0x5580e811c388 <unknown>\nE       #3 0x5580e811ad52 <unknown>\nE       #4 0x5580e811af32 <unknown>\nE       #5 0x5580e81324ca <unknown>\nE       #6 0x5580e81982ef <unknown>\nE       #7 0x5580e8184932 <unknown>\nE       #8 0x5580e8197b94 <unknown>\nE       #9 0x5580e8184823 <unknown>\nE       #10 0x5580e815a08a <unknown>\nE       #11 0x5580e815b1e5 <unknown>\nE       #12 0x5580e83fbe5d <unknown>\nE       #13 0x5580e8415061 <unknown>\nE       #14 0x5580e83fdbb5 <unknown>\nE       #15 0x5580e8415e08 <unknown>\nE       #16 0x5580e83f12f0 <unknown>\nE       #17 0x5580e8431988 <unknown>\nE       #18 0x5580e8431b08 <unknown>\nE       #19 0x5580e844b5dd <unknown>\nE       #20 0x7f4a3fe17609 <unknown>\n\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:229: InvalidArgumentException"}, "start": 1699269496468, "stop": 1699269496815, "uuid": "f90bc643-53b7-408d-a776-e5155c8bc395", "historyId": "18dbf08e03f35faeb303f1b0a2dd21b8", "testCaseId": "18dbf08e03f35faeb303f1b0a2dd21b8", "fullName": "test_home#test_web", "labels": [{"name": "suite", "value": "test_home"}, {"name": "host", "value": "DESKTOP-DVATKHC"}, {"name": "thread", "value": "5280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_home"}]}